docs/shop_administration/customer_portal.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Customer Portal allows your clients to create and manage business account for their company.
+---
+
+# Customer Portal account [[% include 'snippets/experience_badge.md' %]]
+
+Customer Portal allows you to create and manage your business account.
+With this feature, you can edit your organization information,
+invite and view members and check your order history.
+
+![Customer Portal Dashboard](img/cp_dashboard_customer_portal.png)
+
+In the dashboard, you will find a sales representative of your company and a brief summary of your order history.
+For a detailed list of your order history, go to **Pending Order** and **Past Orders** sections.
+
+## Manage members
+
+To view and manage members of your company, go to the **Members** section.
+There you can:
+
+- change the status of each member
+- change their role
+- edit their basic information
+
+![List of members](img/cp_members_portal.png)
+
+To invite new members to your organization, select **Invite members**.
+Then, in a pop-up window fill out email addresses one by one,
+or use drag and drop to upload a file with a list of emails.
+Assign a role to each new member of your team from a drop-down list.
+Click **Send** to send out invitation emails.
+
+![Invite members](img/cp_invite_members_portal.png)
+
+Invited users will receive an email message with a registration link.
+With it, they can register and create their account in the Customer Portal.
+
+In the **Invitation** tab, you can find a list of all invitations,
+registration links and the option to re-send invitations, if needed.
+
+![List of invitations](img/cp_invitations_list_portal.png)
+
+## Address book
+
+If you want to add a new billing address or change the default one, you can do it in the **Address Book** section.
```

Comment:
I think this is similar to what I mentioned before, there is only one billing address (that is required during company creation) so there is no way to add a new one.

Reply:
corrected in https://github.com/ezsystems/user-documentation/pull/151